src/Microsoft.Framework.FileSystemGlobbing/Internal/PathSegments/CurrentPathSegment.cs

```diff
@@ -1,10 +1,14 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+
 namespace Microsoft.Framework.FileSystemGlobbing.Internal.PathSegments
 {
     public class CurrentPathSegment : IPathSegment
     {
+        public bool ProducesStem { get { return false; } }
```

Comment:
Can we find a better name for this property (especially since it is public)? One that comes to my mind is `CanProduceStem`...